### PR TITLE
feat([CTS-10271]): added customisation for java version

### PR DESCRIPTION
**Pull request by vasyl-pryimak**

### CODE REVIEW

**Review by David-Pas — APPROVED** [review]

**Review by dmitry-kokora — APPROVED** [review]

File: .github/workflows/reusable-cd.yml [hunk 1]
```diff
@@ -51,8 +51,7 @@ jobs:
       - name: Prepare ENV vars
         id: setupVars
         run: |
-
-          if [[ "$ENV" =~ ^(dev|qa|uat|staging|prod)$ ]]; then
+          if [[ "$ENV" =~ ^(dev|dev-us|qa|uat|staging|prod|prod-us)$ ]]; then
```
Comment (povisenko) [inline .github/workflows/reusable-cd.yml:54]: I think that it's bad idea
extend the build with AWS region to deploy

**Review by povisenko — CHANGES_REQUESTED** [review]

File: .github/workflows/reusable-cd.yml (hunk 1 above)
Comment (povisenko) [reply .github/workflows/reusable-cd.yml:54]: with the default one to the UK

Comment (dmitry-kokora) [reply .github/workflows/reusable-cd.yml:54]: We discussed this possibility with @vasyl-pryimak , but we did not have time to implement it

Comment (vasyl-pryimak) [reply .github/workflows/reusable-cd.yml]: added ecsClusterName customisation parameter

**Review by povisenko — APPROVED** [review]
looks good now
the only thing is that PR contains 2 features: custom java version and ECS cluster name would prefer it to be as a separate PR/commits next time

**Comment by povisenko** [comment]
@vasyl-pryimak please squash as two commits and after that rebase onto main

**Pull request merged**